ajax/libs/simplebar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "simplebar",
+  "filename": "simplebar.js",
```

Comment:
Let's use `simplebar.min.js` We use minified file as the `filename`
Although I saw `simplebar.js` seem to be minified, I find the file still can be minified by the tool.
So we still need to modify this part.

Sorry for my carelessness.